app/routes/course_type.py

```diff
@@ -0,0 +1,46 @@
+from fastapi import APIRouter
```

Comment:
Missing module DocString.
Please add a todo to indicate that this file still needs work.